systems/plants/CMakeLists.txt

```diff
@@ -63,10 +63,13 @@ if (eigen3_FOUND)
   add_rbm_mex(getCMMmex)
   add_rbm_mex(findKinematicPathmex)
   add_rbm_mex(geometricJacobianmex)
+  add_rbm_mex(doKinematicsNewmex)
   add_rbm_mex(forwardKinVmex)
   add_rbm_mex(forwardKinPositionGradientmex)
   add_rbm_mex(centerOfMassmex)
   add_rbm_mex(centroidalMomentumMatrixmex)
+  add_rbm_mex(massMatrixmex)
```

Comment:
i think that we should seriously consider making only a single entry mex entry point into the rbm class.  this list (and the associate compile time) is getting long...